spec/display_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'display'
+
+RSpec.describe Display do
+
+  before :each do
```

Comment:
You will get different opinions from different people. But I'm of the view that before each blocks are an anti-pattern. They usually are required when you have a lot of setup that needs to be done. Which, usually, is a sign that you're trying to test too much.
In this case you can simply instantiate within the it block.

Reply:
Thanks Matt, I'll do this within the context / describe blocks mentioned in your other comment below